README.md

```diff
@@ -1,2 +1,149 @@
 # Ecommerce
-A Django project to build an online bookstore
+A Django project to build an online bookstore. Build with Django, Bootstrap, and JQuery
+
+# Quick-start guide
+
+This is a quick rundown of necessary steps to set up the project for both development and production. This dedicated for the more experienced developers, and they will be described in more detail below.
+
+- Create a virtual environment (`py -m venv ./venv` to create a new folder on the current directory called venv and then build the env there)
+- Enable it the virtual environment (if using the above example, run `venv/Scripts/Activate`)
+- Download all the packages listed in requirements.txt (`pip install -r requirements.txt)
```

Comment:
I can't run the `manage.py` commands with these requirements. `mod_wsgi` is missing and should probably be included in the requirements. I tried to install it but depends on some system packages 
```
The 'apxs' command appears not to be installed or is not executable. Please check the list of prerequisites in the documentation for this package and install any missing Apache httpd server packages
```
This should be included in pre-requesites

Reply:
This issue was probably caused because I forgot to remove `mod_wsgi.server` from the `INSTALLED_APPS` list after I switched the deployment method from Apache to Waitress. Try running the commands now that it's been removed.